statefun-flink/statefun-flink-core/src/main/java/org/apache/flink/statefun/flink/core/state/PersistedStates.java

```diff
@@ -26,14 +26,31 @@
 import java.util.stream.Stream;
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
+import org.apache.flink.statefun.sdk.FunctionType;
 import org.apache.flink.statefun.sdk.annotations.Persisted;
 import org.apache.flink.statefun.sdk.state.PersistedAppendingBuffer;
 import org.apache.flink.statefun.sdk.state.PersistedTable;
 import org.apache.flink.statefun.sdk.state.PersistedValue;
 
-final class PersistedStates {
+public final class PersistedStates {
```

Comment:
I think that this class should reflect the fact that it reflectively looks for states.
i.e. `ReflectiveStateBinder`.

Reply:
I'd like to avoid naming this class a `*StateBinder` as well, to differentiate it from an actual `StateBinder` implementation.

I chose to rename the static method to `findReflectivelyAndBind` instead.